app/views/variable_lookup_keys/welcome.html.erb

```diff
@@ -0,0 +1,11 @@
+<% content_for(:title, _("Smart Variables")) %>
+<div class="blank-slate-pf">
+  <div class="blank-slate-pf-icon">
+    <%= icon_text("info-circle", "", :kind => "fa") %>
+  </div>
+  <h1><%= _('Smart Variables') %></h1>
+  <p><%= _("Smart variables are a tool to provide global parameters (key/value data), normally to your Puppet ENC,
+           depending on a set of rules. They are intended to be a stepping stone to full parameterized classes,
+           when the class hasn’t been parameterized, or in special cases when a global parameter is desired") %></p>
```

Comment:
https://github.com/theforeman/foreman/pull/4475#discussion_r112650762 is unfixed, missing full stop.